### PR TITLE
163 deprecated conjur v4

### PR DESCRIPTION
**Pull request by sgnn7**
### What does this PR do?

- Deprecates support for Conjur EE v4 in READEME
- Adds link to our module on puppetforge

### What ticket does this PR close?
Connected to #163 
Connected to #174 

#### Change log
- [ ] The CHANGELOG has been updated, or
- [x] This PR does not include user-facing changes and doesn't require a CHANGELOG update

#### Test coverage
- [ ] This PR includes new unit and integration tests to go with the code changes, or
- [x] The changes in this PR do not require tests

#### Documentation
- [x] Docs (e.g. `README`s) were updated in this PR, and/or there is a follow-on issue to update docs, or
- [ ] This PR does not require updating any documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,10 +24,12 @@
 ## Description
 
 This is the official Puppet module for [Conjur](https://www.conjur.org), a robust
-identity and access management platform. This module simplifies the operations involved in 
+identity and access management platform. This module simplifies the operations involved in
 establishing a Conjur host identity and allows authorized Puppet nodes to fetch
 secrets from Conjur.
 
+You can find our official distributable releases on puppetforge under [`cyberark/conjur`](https://forge.puppet.com/cyberark/conjur).
```
Comment (uCatu) [inline README.md:31]: 'puppetforge' or 'Puppet Forge'?

File: README.md [hunk 2]
```diff
@@ -41,7 +42,10 @@ This module requires that you have:
   module. Supported versions:
   - Conjur OSS v1+
   - DAP v10+
-  - Conjur Enterprise v4.9+
+
+**Note that Conjur Enterprise v4 support is deprecated. If you are still using Conjur v4,
```
Comment (uCatu) [inline README.md:46]:
```suggestion
**Note that Conjur Enterprise v4 support is deprecated. If you are still using this version,
```

File: README.md (hunk 1 above)
Comment (uCatu) [inline README.md:31]:
```suggestion
You can find our official distributable releases on [`Puppet Forge cyberark/conjur`](https://forge.puppet.com/cyberark/conjur).
```

Comment (sgnn7) [reply README.md]: Fixed

File: README.md (hunk 2 above)
Comment (sgnn7) [reply README.md]: Fixed

**Pull request merged**